src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -100,7 +104,21 @@
             HelpText = "Execution traces, expressed as a semicolon-delimited list, that " +
                        "should be emitted to the console and log file (if appropriate). " +
                        "Valid values: ScanTime.")]
-        public virtual IEnumerable<string> Traces { get; set; }
+        public virtual IEnumerable<string> Trace { get; set; }
+
+        private DefaultTraces? defaultTraces;
+        public DefaultTraces Traces
```

Comment:
>Traces [](http://example.com/codeflow?start=29&length=6)

This pattern may be worth pursuing aggressively, i.e., some class surface area for interacting directly with the commandline library vs. reconstructed data more suitable for direct consumption by the engine.

I keep wondering whether options could be merged entirely with the context object...